src/site/content/en/blog/when-to-use-local-https/index.md

```diff
@@ -10,6 +10,8 @@ tags:
   - security
 ---
 
+Also see: [How to use HTTPS for local development](/how-to-use-local-https).
```

Comment:
FWIW I think this one will have the most impact on pointing readers to the other post.

Reply:
I see, are you suggesting to remove this?

Reply:
(I tend to think the *How* article is more useful that the *When*, so if readers navigate to *How* I'm feeling OK with this)

Reply:
No no, the opposite, I think this link placement best solves the issue of readers missing the other post.